web/src/main/java/org/openmrs/web/controller/concept/ConceptDatatypeListController.java

```diff
@@ -101,7 +101,7 @@ protected ModelAndView onSubmit(HttpServletRequest request, HttpServletResponse
 			if (!success.toString().equals("")) {
 				httpSession.setAttribute(WebConstants.OPENMRS_MSG_ATTR, success.toString());
 			}
-			if (!error.equals("")) {
+			if (!error.toString().equals("")) {

```

Comment:
@harsha89, this looks better if it is to be: !("".equals(error.toString())), same reason you have just given me, to avoid null exception